### ISSUE TITLE
Reinstalling does not reinstall

### ISSUE THREAD

**Issue by shoffmeister** [WhitewaterFoundry/Fedora-Remix-for-WSL#191]
I am attempting to reinstall Fedora Remix for WSL - i.e. really eradicate whatever is there and start again with a shiny new, empty, default WSL'ed Fedora Remix for WSL (I wrecked my image).

Alas, reinstalling does not reinstall - and, honestly, I don't know what it does. Reinstall the _app_?

For reinstalling as I understand it,
* `wsl --shutdown` for good measure (`--terminate fedoraremix` should suffice, though)
* `wsl --unregister fedoraremix`
* Uninstall Fedora Remix for WSL through Windows control panel
* Reinstall Fedora Remix for WSL

**Comment by crramirez** [comment]
Yes, you are right; it only reinstalls the app. This is how all WSL distros work. but it is a good point and I'll take note to see if we can do anything from the app side.

For reinstalling, just the first two steps are enough. And in most cases, the second one is enough.

Sorry for the delayed answer, I just saw your issue.
